src/classes/MisSubcountyMarketPrices.cls

```diff
@@ -0,0 +1,851 @@
+            SELECT 
+                AVG(A_Kg_of_Mazie_grain_at_retail_price__c)retail, 
+                AVG(A_Kg_of_Mazie_grain_at_wholesale_price__c)wholesale
+            FROM 
+                X3_0_Masindi_Maize__c 
+            WHERE 
+                X3_0_Market_Masindi__r.Date_of_capture__c = yesterday ];   
+                
+        Menu_Item__c maizeMenuItem = [
+            SELECT  
+                Label__c, 
+                Parent_Item__r.Label__c 
+            FROM 
+                Menu_item__c 
+            WHERE 
+                Parent_Item__r.Label__c = : marketName
+            AND 
+                Label__c='Maize'];
+                
+        if(String.valueOf(maizeResult[0].get('retail')) != null && String.valueOf(maizeResult[0].get('wholesale')) != null){
+            Integer averageRetailPrice = Integer.valueOf(maizeResult[0].get('retail'));
+            Integer averageWholesalePrice = Integer.valueOf(maizeResult[0].get('wholesale'));
+            
+            maizeMenuItem.Content__c = 'Average Retail Price '+averageRetailPrice+' Per Kg'+
+                '\N Average Wholesale Price '+ averageWholesalePrice+' Per Kg';
+            update maizeMenuItem;
+            //return averageRetailPrice + ' '+ averageWholesalePrice;
+        } 
+    }
+    
+    public static void computeMasindiSunflowerPriceAverages(String marketName) {
+        //computes average retail and wholesale prices per Kg of Sunflower
+        AggregateResult[] sunflowerResult = [
+            SELECT 
+                AVG(A_Kg_of_Sunflower_at_retail_price__c)retail, 
+                AVG(A_Kg_of_Sunflower_at_wholesale_price__c)wholesale
+            FROM 
+                X3_0_Masindi_Sunflower__c 
+            WHERE 
+                X3_0_Market_Masindi__r.Date_of_capture__c = yesterday ]; 
+                
+            Menu_Item__c sunflowerMenuItem = [
+            SELECT 
+          //      Id, 
+                Label__c, 
+                Parent_Item__r.Label__c 
+             //   Name 
+            FROM 
+                Menu_item__c 
+            WHERE 
+                Parent_Item__r.Label__c = : marketName
+            AND 
+                Label__c='Sunflower'];
+             
+        if (String.valueOf(sunflowerResult[0].get('retail')) != null && String.valueOf(sunflowerResult[0].get('wholesale')) != null){   
+            Integer averageRetailPrice = Integer.valueOf(sunflowerResult[0].get('retail'));
+            Integer averageWholesalePrice = Integer.valueOf(sunflowerResult[0].get('wholesale'));
+        
+            sunflowerMenuItem.Content__c = 'Average Retail Price '+ averageRetailPrice +' Per Kg'+
+                '\N Average Wholesale Price '+ averageWholesalePrice +' Per Kg';
+        
+            update sunflowerMenuItem;
+        }    
+    }
+    
+    public static void computeMasindiCassavaPriceAverages(String marketName) {
+        //computes average retail and wholesale prices per Kg of Cassava
```

Comment:
its good you are commenting. but please,if you are generally talking about what the method is doing, put it above the method definition